src/elements/link-list/src/index.tsx

```diff
@@ -24,25 +24,29 @@ export const LinkList: React.FC<ListLinkProps> = ({
 }) => {
   return (
     <div className={className} sx={{ variant: styles(variant) }}>
-      <div
-        sx={{
-          display: 'flex',
-          alignItems: 'center',
-          variant: styles(variant, 'header')
-        }}
-      >
-        {icon}
-        <Styled.h3
-          as="h2"
+      {(title || icon) && (
+        <div
```

Comment:
I know this isnt new but as this is the title of the link list shouldn't this be a <header> element

Reply:
@jonathan-fielding it's not just the tile, it's also the icon